prime-router/src/main/kotlin/cli/LookupTableCommands.kt

```diff
@@ -567,8 +574,10 @@ class LookupTableCreateCommand : GenericLookupTableCommand(
             return
         }
 
-        // Output the data for review.
-        if (!silent) {
+        // Output the data for review specified.
+        val isLargeTable = inputData.size > 50 || inputData[0].keys.size > 7
```

Comment:
50 and 7 should be assigned somewhere, kinda magic numbers.

Reply:
You are right.  Let me think about this.  Will try to maybe use one value instead